src/shared/sam/debugger/samDebugSession.ts

```diff
@@ -219,14 +234,22 @@ export class SamDebugSession extends DebugSession {
         // this.sendResponse(response);
     }
 
-    protected dataBreakpointInfoRequest(response: DebugProtocol.DataBreakpointInfoResponse, args: DebugProtocol.DataBreakpointInfoArguments): void {
-    }
+    protected dataBreakpointInfoRequest(
+        response: DebugProtocol.DataBreakpointInfoResponse,
+        args: DebugProtocol.DataBreakpointInfoArguments
+    ): void {}
 
-    protected setDataBreakpointsRequest(response: DebugProtocol.SetDataBreakpointsResponse, args: DebugProtocol.SetDataBreakpointsArguments): void {
+    protected setDataBreakpointsRequest(
+        response: DebugProtocol.SetDataBreakpointsResponse,
+        args: DebugProtocol.SetDataBreakpointsArguments
```

Comment:
I would also be in favor of adding trailing comma in param lists, but I assume I'm the only one. So that's why I chose `es5` instead of `all`. Note that `all` won't add trailing commas to single-line lists.